docs/user/tutorial/10-log-in.rst

```diff
@@ -2,5 +2,58 @@
 Log in
 ######
 
-.. todo:: login instructions for the site on Aldryn
+The fastest way to setup a pre-configured django-cms website is using Aldryn.
 
+.. image:: /user/tutorial/images/aldryn-vertical_black.png
+    :alt: Aldryn logo
+    :align: center
+    :width: 260px
+
+.. note::
+
+    **Aldryn**
+
+    Aldryn is an easy-to-use and developer-friendly content management platform.
+    Aldryn has been designed to meet the needs of:
+        - **web content creators**, who require an efficient and reliable way to produce content
+        - **agencies**, who work with multiple clients and content creators, and need to manage them all
+        - **web developers**, who implement designs and create applications, and need a low-friction workflow for their deployment
+
+    For more details, please check http://aldryn.com
+
+
+.. note::
+
+    **How to use Aldryn**
+
+    In this tutorial we are not going to cover how to use Aldryn.
+    For this purpose Aldryn provides a help section with plenty of tutorials.
+
+    Please visit: http://aldryn.com/help
+
+Please proceed with the following steps once the project has been setup and deployed.
+
+1. To login to your new website, hit the |open-site| link on the dashboard.
+
+    .. |open-site| image:: /user/tutorial/images/open_site.png
+       :alt: Open site
+
+    .. image:: /user/tutorial/images/aldryn_dashboard.png
+       :alt: Aldryn Dashboard
+
+2. Click on *sign in with Aldryn*
+
+    .. image:: /user/tutorial/images/aldryn_signin.png
+       :alt: Aldryn Sign in
+
+3. Fill out the form with your credentials and hit *sign in*
```

Comment:
Surely this won't be necessary if the user has just deployed the site? It will already have logged them in, in the previous step, no?